Add Ceres products to SORA docs (#225)
sora-xor/sora-docs: src/ceres/demeter-farming.md

---
title: "Demeter Farming | SORA Docs"
head:
  - - meta
    - name: description
      content: "The Demeter Farming platform is the first of its kind on the SORA network."
  - - meta
    - name: keywords
      content: "Ceres, Demeter, DeFi, farming, staking, farming-as-a-service, Polkaswap, SORA network"
---

> Demeter (DEO) is the native token of Demeter farming platform and DEO Arena Web3 game. In the Ceres ecosystem, Demeter is the first airdrop with its own purpose.

Reviewer: ```suggestion
Demeter (DEO) is the native token of the Demeter farming platform and DEO Arena Web3 game. In the Ceres ecosystem, Demeter is the first self-purpose airdrop.
```

# Demeter Farming

> [Demeter Farming](https://farming.deotoken.io/) platform is the first of its kind on the SORA network. 

Reviewer: ```suggestion
The [Demeter Farming](https://farming.deotoken.io/) platform is the first of its kind on the SORA network. 
```

> The unique feature of this platform is the user's ability to farm with the same liquidity in multiple pools. 

Reviewer: ```suggestion
This platform's unique feature is giving the user the ability to farm using the same liquidity in multiple pools. 
```

> The platform also provides Farming-as-a-Service, which means that every project can create its farming/staking pools without coding.

Reviewer: ```suggestion
The platform also provides Farming-as-a-Service, meaning every project can create farming/staking pools without coding.
```

---
title: "Demeter Farming | SORA Docs"
head:
  - - meta
    - name: description
      content: "The Demeter Farming platform is the first of its kind on the SORA network."
  - - meta
    - name: keywords
      content: "Ceres, Demeter, DeFi, farming, staking, farming-as-a-service, Polkaswap, SORA network"
---

> 

Reviewer: ```suggestion
# Demeter (DEO)
```
